collector/monitors/validators_repository_v1.go

```diff
@@ -77,5 +77,6 @@ func (r *V1ValidatorsRepository) GetValidatorInfo(ctx context.Context, address s
 		Moniker:        validatorInfoResponse.GetPayload().Result.Description.Moniker,
 		PubKey:         *validatorInfoResponse.GetPayload().Result.ConsensusPubkey,
 		CommissionRate: commissionRate,
+		Jailed:         *validatorInfoResponse.GetPayload().Result.Jailed,
```

Comment:
The same field forgotten for V2 implementation. GetValidatorInfo same for both implementations, i think we can reuse it.

Reply:
good call, fixed